packages/lib/src/index.ts

```diff
@@ -1,28 +1,7 @@
 import * as typeIs from './typeIs';
 import * as testHelpers from './testHelpers';
 
-/**
- * Utility for making it easier to couple a React Component to a css selector.
- * Useful when writing css selectors that rely on interactivity, i.e. :hover.
- * Example:
- *  const checkBoxWrapper = createDataProp('checkbox-wrapper');
- *  // Used as selector:
- *  css`&:hover ${checkboxWrapper.selector} { }`
- *  // Used on React Component
- *  <div {...checkboxWrapper.prop} />
- * @param {string} name Name of element we want to reference.
- */
-export function createDataProp(name: string) {
-  // ts:emit throws an error if this is not typed. Usually this can be inferred.
-  const prefix = 'data-leafygreen-ui' as string;
-
-  return {
-    prop: {
-      [prefix]: name,
-    },
-    selector: `[${prefix}="${name}"]`,
-  };
-}
+export { typeIs, testHelpers };
```

Comment:
Can do new export syntax now!

Reply:
Is there a way to neatly do this: `typeIs` and  `testHelpers` are imported like so:

```js
import * as typeIs from './typeIs';
```

AFAIK we can't do:
```js
export * as typeIs from './typeIs';
```

Reply:
I believe you can 😱  https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export

Reply:
Oh hmm... it was yelling at me previously for doing this but it seems to be working now! Will do.

Reply:
Actually, the current version of TS is still yelling. 
```sh
MongoDB/leafygreen-ui [PD-567-icon-button-accessibility●] » yarn ts
yarn run v1.16.0
$ tsc --build tsconfig.json

packages/lib/src/index.ts:1:10 - error TS1005: 'from' expected.
1 export * as typeIs from './typeIs';

packages/lib/src/index.ts:1:13 - error TS1005: ';' expected.
1 export * as typeIs from './typeIs';

packages/lib/src/index.ts:1:20 - error TS1005: ';' expected.
1 export * as typeIs from './typeIs';

packages/lib/src/index.ts:1:25 - error TS1005: ';' expected.
1 export * as typeIs from './typeIs';
```
Leaving for now and we can try again with the next version of TS if that's alright.

Reply:
WFM